src/AnnouncementBubble/NormalAnnouncementBubble.less

```diff
@@ -155,8 +155,20 @@ button.NormalAnnouncementBubble--deleteMenuItem {
 
 .NormalAnnouncementBubble--translatedLanguages--tooltip {
   .text--semi-bold();
-  padding-left: @size_xs;
-  padding-right: @size_xs;
+  padding-left: @size_2xs;
+  padding-right: @size_2xs;
+  white-space: pre-wrap;
+}
+
+.Button.NormalAnnouncementBubble--translatedLanguages--button {
+  &:focus {
+    outline: 0.1875rem solid @primary_blue_shade_2;
+    transition: none;
+  }
+
+  &:focus:not(:focus-visible) {
```

Comment:
I think `:focus-visible` doesn't work well on Safari, or works differently? I tried using it elsewhere recently and there were issues. 😬 

Just suggest checking it out!

Reply:
I've just tested on Safari and had no issues with these changes in particular. The MessagingThreadHistory is still getting outlined when we don't want it to, but everything is still getting outlined when it's supposed to. So it's a problem of too many outlines
But things are working and looking ok for these changes (and most others besides ThreadHistory!)